src/pages/sepa.js

```diff
@@ -0,0 +1,145 @@
+    this.search = parseLocationSearch(this.props.location);
+  },
+  componentDidMount: function() {
+    if (typeof window === "undefined") return;
+    if (window.document === "undefined") return;
+    var head = window.document.querySelector('head');
+    var script = window.document.createElement('script');
+    script.src = 'https://js.stripe.com/v3/';
+    script.onload = (e) => this.setState({ stripeLoaded: true });
+    head.appendChild(script);
+  },
+  render: function() {
+    return (
+      <div className={'row'}>
+        <Header/>
+        <div className="container">
+          <h4>Donate via SEPA</h4>
+          { this.showLeadinContent() }
+          { this.getSepaForm() }
+        </div>
+        <SmallPrint/>
+        <MozillaFooter/>
+      </div>
+    );
+  },
+  showLeadinContent: function() {
+    if (!this.props.amount || this.state.showChangeForm) {
+      return [<Frequency key={'freq'}/>, <AmountButtons key={'amt'}/>];
+    }
+
+    // TODO: I'm not a fan of using <a href=#> but we'll need a user-tabbable
+    //       element here so we might be able to do a <span> with a tabindex=0
+    //       so that people can tab to it to trigger the freq/amount components.
+    return [
+      <p key={'commit_msg'}>
+        You are committing to a {this.props.frequency} donation of {this.props.amount} euro.
+      </p>,
+      <p key={'change_msg'}>
+        <a href='#' onClick={evt => this.setState({ showChangeForm: true }) }>
+          Click here to change the amount and/or frequency.
+        </a>
+      </p>
+    ];
+  },
+  getSepaForm: function() {
+    if (!this.state.stripeLoaded) {
+      return null;
+    }
+
+    return (
+      <form className="row full">
+          <label htmlFor={'namefield'}>Name</label>
+          <input type="text" id={'namefield'} onChange={e => this.handleName(e)}/>
+
+          <label htmlFor={'ibanfield'}>IBAN</label>
+          <input type="text" id={'ibanfield'} onChange={e => this.handleIBAN(e)}/>
+
+          <div className="submit-button">
+            <button className="donate-submit submit-btn" onClick={e => this.handleSubmit(e)}>Submit</button>
+          </div>
+      </form>
+    );
+  },
+  handleName(e) {
+    let name = e.target.value;
+    this.setState({ name });
+  },
+  handleIBAN(e) {
+    let iban = e.target.value;
+    this.setState({ iban });
+  },
+  handleSubmit(e) {
+    e.preventDefault();
+
+    // See https://stripe.com/docs/sources/sepa-debit#prerequisite
+
+    if (typeof Stripe !== 'undefined') {
+      this.setState({ submitting: true }, () => {
+        let key = process.env.STRIPE_PUBLIC_KEY;
+        let stripe = Stripe(key);
+
+        stripe.createSource({
+          type: 'sepa_debit',
+          sepa_debit: {
+            iban: this.state.iban
+          },
+          currency: 'eur',
+          owner: {
+            name: this.state.name
+          }
+        }).then( result => {
+          console.log(result);
+
+          if (result.error) {
```

Comment:
this should be an object in the form of https://stripe.com/docs/api/node#errors
where you would use the codes in https://stripe.com/docs/sources/sepa-debit#error-codes to determine what kind of error it was. 

You might want to check in with @ScottDowne about how we handle errors for card payments when creating a source object on the front-end (do we show a generic error message, etc.)